addons/fleet/migrations/13.0.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,82 @@
+---Models in module 'fleet'---
+---Fields in module 'fleet'---
+fleet        / fleet.vehicle            / future_driver_id (many2one)   : NEW relation: res.partner
+fleet        / fleet.vehicle            / net_car_value (float)         : NEW
+fleet        / fleet.vehicle            / next_assignation_date (date)  : NEW
+fleet        / fleet.vehicle            / plan_to_change_car (boolean)  : NEW isrelated: related, stored
+fleet        / fleet.vehicle.cost       / company_id (many2one)         : NEW relation: res.company, hasdefault
+fleet        / fleet.vehicle.model      / manager_id (many2one)         : NEW relation: res.users, hasdefault
```

Comment:
Filling it with uid=1 doesn't make sense. Better to empty it (or pre-create the column for avoiding the default filling)